lib/src/excel.dart

```diff
@@ -167,12 +167,13 @@ class ExcelDart {
       sheet.getUsedRange().getEntireColumn().format.autofitColumns();
       sheet.getUsedRange().getEntireRow().format.autofitRows();
 
-      // Sets the sheet as active and syncs context.
+      // Sets the sheet as active.
       sheet.activate();
-      context.sync();
 
       // All operations are complete.
       populateCompleter.complete();
+
+      return context.sync();
```

Comment:
Excel js expects this callback function to return context.sync(). Even though the population still works without it, Excel js will complain in the console.